src/scene/shader-lib/chunks/common/frag/linearizeDepth.js

```diff
@@ -0,0 +1,32 @@
+export default /* glsl */`
```

Comment:
a lot of this is a duplication from the screenDepth.js. Should we find a solution without the duplication?

Reply:
I can't change screenDepth without breaking other projects that may rely on it. I thought about moving all the linearization code to this chunk, and then have screenDepth only be specifically for getting depth at a pixel coordinate, but then I realized a lot of people might be using screenDepthPS in their own chunks to get this functionality and it would break things for them.